.github/workflows/test-core.yaml

```diff
@@ -58,22 +72,23 @@ jobs:
       - uses: magnetikonline/action-golang-cache@v1
         with:
           go-version: ${{env.GO_VERSION}}
-          cache-key-suffix: -compile
+          cache-key-suffix: -core
       - name: Run make dev
         env:
           GOBIN: ${{env.GOROOT}}/bin # windows kludge
         run: |
           make bootstrap
```

Comment:
If we're caching the results of `make bootstrap` in `mods`, do we need this anymore?

Reply:
We do because GOBIN isn't part of the cache - the executables aren't cached across runners, only the GOMODCACHE and GOCACHE contents. https://github.com/magnetikonline/action-golang-cache/issues/1 mentions the possibility of adding GOBIN but hasn't gotten traction. The resolution of https://github.com/actions/setup-go/issues/278 might also impact what happens to the binaries.